robottelo/performance/candlepin.py

```diff
@@ -29,7 +29,7 @@ class Candlepin(object):
     serverhost = conf.properties['main.server.hostname']
 
     @classmethod
-    def get_real_time(result):
+    def get_real_time(cls, result):
```

Comment:
You're not using the `cls` argument in this method. So why bother making this a classmethod? How about making this a staticmethod?